src/models/Studio.ts

```diff
@@ -45,8 +44,13 @@ export function start(filePath: string, port: number = DEFAULT_PORT): void {
   });
 
   const server = createServer((request, response) => {
+    //not all CLI users use npm. Some package managers put dependencies in different weird places
+    //this is why we need to first figure out where exactly is the index.html located 
+    //and then strip index.html from the path to point to directory with the rest of the studio
+    const indexLocation = require.resolve('@asyncapi/studio/build/index.html');
```

Comment:
This is looking for a global install studio right?

Reply:
for any closest
so if you use CLI as local dependency, this one will be picked up